react/src/components/UserCredentialList.tsx

```diff
@@ -0,0 +1,529 @@
+            render: (record) => {
+              return (
+                <Flex direction="column" align="start">
+                  <Typography.Text>
+                    {record.concurrency_used}
+                    <Typography.Text
+                      type="secondary"
+                      style={{
+                        marginLeft: token.marginXXS,
+                        fontSize: token.fontSizeSM,
+                      }}
+                    >
+                      {t('credential.Sessions')}
+                    </Typography.Text>
+                  </Typography.Text>
+                  <Typography.Text
+                    style={{
+                      fontSize: token.fontSizeSM,
+                    }}
+                  >
+                    {record.rate_limit}
+                    <Typography.Text
+                      type="secondary"
+                      style={{
+                        marginLeft: token.marginXXS,
+                        fontSize: token.fontSizeSM,
+                      }}
+                    >
+                      {t('credential.ReqPer15Min')}
+                    </Typography.Text>
+                  </Typography.Text>
+                  <Typography.Text
+                    style={{
+                      fontSize: token.fontSizeSM,
+                    }}
+                  >
+                    {record.num_queries}
+                    <Typography.Text
+                      type="secondary"
+                      style={{
+                        marginLeft: token.marginXXS,
+                        fontSize: token.fontSizeSM,
+                      }}
+                    >
+                      {t('credential.Queries')}
+                    </Typography.Text>
+                  </Typography.Text>
+                </Flex>
+              );
+            },
+          },
+          {
+            key: 'control',
+            title: t('general.Control'),
+            fixed: 'right',
+            render: (record) => {
+              return (
+                <Flex gap={token.marginXS}>
+                  <Button
+                    type="text"
+                    icon={
+                      <InfoCircleOutlined
+                        style={{ color: token.colorSuccess }}
+                      />
+                    }
+                    onClick={() => {}}
+                  />
+                  <Button
+                    type="text"
+                    icon={
+                      <SettingOutlined style={{ color: token.colorInfo }} />
+                    }
+                    onClick={() => {}}
+                  />
+                  {activeType === 'inactive' && (
+                    <Tooltip title={t('credential.Active')}>
+                      <Popconfirm
+                        title={t('credential.ConfirmUpdateStatusToActive')}
+                        description={record.user_id}
+                        okType="danger"
```

Comment:
```suggestion
```

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/54812d6f-7342-4c34-8ffb-d87f193c852d.png)

It doesn't need to be a danger type.

Reply:
In my opinion, activating/inviting another user's keyfair is a behavior that can limit their overall WebUI experience, so I think it's a good idea to give them a visual warning via danger :)

Reply:
But in the user tab, we use the primary type and OK message. As I think, inactivating is a kind of deleting and activating is recovery to fix the error. So I don't think we need to use danger type for activating.
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/995b0aef-02c4-4ce6-8f88-6f3584a3ac71.png)

Reply:
Oh you meant activating, I misunderstood, I've fixed it for activating.